app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -73,7 +83,15 @@ def discover_queue(username, password, zone = nil)
     private
 
     def discover_from_queue(username, password)
-      discover(username, MiqPassword.decrypt(password))
+      discover(username, '')
```

Comment:
Sure thing. But I do not think neither username or password should be passed, don't you think?